components/dialog/stories/dialog.stories.js

```diff
@@ -65,17 +110,159 @@ export default {
 	},
 };
 
+const ExampleContent = Typography({
+	semantics: "body",
+	size: "m",
+	content: [
+		"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Auctor augue mauris augue neque gravida. Libero volutpat sed ornare arcu. Quisque egestas diam in arcu cursus euismod quis viverra. Posuere ac ut consequat semper viverra nam libero justo laoreet. Enim ut tellus elementum sagittis vitae et leo duis ut. Neque laoreet suspendisse interdum consectetur libero id faucibus nisl."
+	]
+});
+
+const ExampleButtons = [{
+	variant: "secondary",
+	treatment: "outline",
+	label: "Remind me later"
+}, {
+	treatment: "fill",
+	label: "Enable",
+	variant: "accent"
+}];
```

Comment:
None of the dialogs on the docs site currently have buttons, so I added them in any of the examples that _weren't_ dismissible.

Reply:
Removing buttons in favor of that work being done in a different PR.